specs/experimental/engine_queue_block_derivation.md

```diff
@@ -0,0 +1,47 @@
+# Engine Queue Block Derivation
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Motivation](#motivation)
+- [Solution](#solution)
```

Comment:
So far the specs docs are written differently, less so in the form of a proposal. I like the contextual information, but maybe a `Rationale` section at the bottom of the doc that combines the `Overview` and `Motivation`, and then the `Solution` as the primary content, makes more sense for consistency with other specs. We can pivot to this new format though, if we plan to write more docs this way, if things go more in the direction of an EIP type document. cc @tynes

Reply:
Removed the Overview and added a Rationale section about the why